### PR TITLE
Quote Block, Hero Block bug fix

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix 

Test URLs:
- Before: https://main--hsf-commonmoves--hlxsites.hlx.live/drafts/piyush/design-1-quote-hero
- After: https://quote-changes--hsf-commonmoves--hlxsites.hlx.live/drafts/piyush/design-1-quote-hero

### CODE REVIEW

**Review by bstopp — COMMENTED** [review]
Shouldn't this only be applied if there is an agent profile to display?

**Comment by jindaliiita** [comment]
@bstopp Can you review this PR again

File: styles/styles.css [hunk 1]
```diff
@@ -552,6 +552,12 @@ form button[type="submit"]:hover {
   }
 }
 
+@media screen and (min-width: 992px) {
```
Comment (bstopp) [inline styles/styles.css:555]: We don't have this breakpoint. it's only 600, 900, 1200.

File: blocks/hero/search/search.css [hunk 2]
```diff
@@ -148,3 +148,9 @@
     padding: 10px;
   }
 }
+
+@media screen and (min-width: 1200px) and (max-width: 1400px) {
```
Comment (bstopp) [inline blocks/hero/search/search.css:152]: Why do we stop at 1400px?

**Review by bstopp — CHANGES_REQUESTED** [review]

**Review by bstopp — APPROVED** [review]

**Pull request merged**